demo/src/App.jsx

```diff
@@ -63,6 +64,16 @@ const App = () => (
           <option value="2" disabled>Option 2</option>
           <option value="3">Option 3</option>
         </Form.Select>
+        <Card.Item>
```

Comment:
And for statuses 
```
<Card status="warning" /> <Card status="error" />
```
etc...

using these props our React component internally can add the required classNames without the consumer having to be concerned about them.  In our components wherever possible prefer props over className.  We provide a className hook for consumers to add their own styling if they desire to extend on Nebula.